docs/version0.33.md

```diff
@@ -50,6 +51,10 @@ This was already required for OpenJDK 17 and is now also required from 11.0.16 t
 
 Linux builds for all platforms now use gcc 10.3 instead of gcc 7.5. See the list of [build environments](openj9_support.md#build-environments).
 
+### The maximum number of compilation threads that are used by the JIT compiler increased
+
+The maximum number of compilation threads increased from 7 to 15 if the VM is running with the JITServer feature enabled (JIT and JITServer client). The maximum number of compilation threads on the JITServer server is increased to 999.
+
```

Comment:
The text is not quite correct  - the 15 limit also applies when the JITServer feature is not enabled. We could use a link too. I suggest something like the following:

"The maximum number of JIT compilation threads is increased from 7 to 15. If the JITServer feature is enabled, the maximum number of JIT compilation threads is increased further on the server, to 999. You can set the number of compilation threads to be used by specifying the `-XcompilationThreads` command line option[link].”